apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/NamespaceService.java

```diff
@@ -292,6 +296,9 @@ private void fillAppNamespaceProperties(NamespaceBO namespace) {
 
         deletedItem.setDeleted(true);
         ItemDTO deletedItemDto = new ItemDTO();
+        if (deletedItemDTOs.containsKey(key)) {
+          deletedItemDto = deletedItemDTOs.get(key);
+        }
```

Comment:
Combine these lines into

```java
ItemDTO deletedItemDto = deletedItemDTOs.computeIfAbsent(key, k -> new ItemDTO());
```

to reduce unnecessary object creation (`ItemDTO`) every time